faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -50,10 +50,16 @@ class ClientSpec extends FlatSpec with Matchers with BeforeAndAfterAll {
   val PageRefs = Field("data").to[Seq[RefV]]
 
   def await[T](f: Future[T]) = Await.result(f, 5.second)
+  def ready[T](f: Future[T]) = Await.ready(f, 5.second)
+
+  def dropDB(): Unit =
+    ready(rootClient.query(Delete(Ref(s"databases/$testDbName"))))
```

Comment:
That is the intention because it's being called on both setup and teardown. If the db doesn't exists during setup, it'll be created.